### PR TITLE
New PR for Cortex XDR - Malware investigation.

### PR DESCRIPTION
**Pull request by ssokolovich**
Added PBs + Readme files + Update RN

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
New PR for Cortex XDR - Malware investigation.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation.yml [hunk 1]
```diff
@@ -0,0 +1,2618 @@
...
+        "height": 5785,
+        "width": 2280,
+        "x": -190,
+        "y": -1090
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation.yml:2618]: 6.5

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling.yml [hunk 2]
```diff
@@ -0,0 +1,561 @@
+contentitemexportablefields:
+  contentitemfields:
+    propagationLabels:
+    - all
+id: Cortex XDR - False Positive Incident Handling
+inputs:
+- description: Add comment to close an incident on the Microsoft Defender For Endpoint side
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling.yml:7]: defender

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* xdr-allowlist-files
+* setIndicators
+* closeInvestigation
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Comment | Add comment to close an incident on the Microsoft Defender For Endpoint side | XSOAR Incident #${incident.id} | Optional |
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md:25]: Defender

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling_README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* closeInvestigation
+* jira-create-issue
+* xdr-blocklist-files
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Comment | Add comment to close an incident on the Microsoft Defender For Endpoint side | XSOAR Incident #${incident.id} | Optional |
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling_README.md:28]: defender

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml [hunk 5]
```diff
@@ -0,0 +1,1149 @@
...
+        "height": 1430,
+        "width": 2670,
+        "x": -3070,
+        "y": -160
+      }
+    }
+  }
+tests:
+- Test XDR Playbook
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml:1148]: 6.5

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml [hunk 6]
```diff
@@ -0,0 +1,706 @@
...
+      - '22'
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ''
+      id: dd59e28d-bdf5-43a0-8a9e-931e994addd3
+      iscommand: false
+      name: Check If there is a mite technique
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml:676]: MITRE

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml [hunk 7]
```diff
@@ -0,0 +1,706 @@
...
+      description: Check If there is a mite technique
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml:677]: MITRE

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml [hunk 8]
```diff
@@ -0,0 +1,706 @@
...
+        "height": 1435,
+        "width": 1290,
+        "x": -30,
+        "y": 370
+      }
+    }
+  }
+tests:
+- Test XDR Playbook
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment.yml:705]: 6.5

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment_README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+| Endpoint.OS | The endpoint's operating system. | unknown |
+| Endpoint.OSVersion | The endpoint's operating system's version. | unknown |
+| Endpoint.IPAddress | The endpoint's IP address. | unknown |
+| Endpoint.Status | The health status of the endpoint. | unknown |
+| Endpoint.MACAddress | The endpoint's MAC address. | unknown |
+| Endpoint.Vendor | The integration name of the endpoint vendor. | unknown |
+
+## Playbook Image
+---
+![Cortex XDR Malware - Incident Enrichment](../doc_files)
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Incident_Enrichment_README.md:50]: path

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Investigation_And_Response_README.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* 470ceb33-1487-47a0-8337-a6c50149f385
+* Command-Line Analysis
+* Cortex XDR - False Positive Incident Handling
+* Dedup - Generic v4
+* 5d24f937-9988-4647-8247-7415e343b581
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Malware_-_Investigation_And_Response_README.md:11]: ID

File: Packs/CortexXDR/ReleaseNotes/4_6_0.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+* Host fields (Host ID)
+* Attacker fields (Attacker IP, External host)
+* MITRE techniques
+* File hash (currently, the playbook supports only SHA256)  
+
+Note: The playbook inputs enable manipulating the execution flow; read the input descriptions for details. (Available from Cortex XSOAR 6.5.0).
+##### New: **Cortex XDR Malware - Incident Enrichment**
+- This playbook aims to enrich the Cortex XDR incident. The enrichment is done on: the involved endpoint, Mitre technique ID information, and sets the 'Malware-Investigation and Response' layout. (Available from Cortex XSOAR 6.5.0).
+##### New: **Cortex XDR - False Positive Incident Handling**
+- This Playbook handles a false-positive incident closure for Cortex XDR - Malware investigation. (Available from Cortex XSOAR 6.2.0).
```
Comment (David-BMS) [inline Packs/CortexXDR/ReleaseNotes/4_6_0.md:18]: 6.5.0

**Review by David-BMS — APPROVED** [review]
@michalgold @altmannyarden Are you planning to review as well?

**Comment by julieschwartz18** [comment]
Doc review complete

**Pull request merged**